packages/toolkit/.changes/next-release/Bug Fix-c45df2a7-e8f0-488a-815c-0883ab3919b9.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "AWS Lambda: Added missing strings in guided walkthrough"
```

Comment:
FYI: the reminder comment is advisory. For fixes to unreleased features, we don't need a changelog item.

Reply:
removed